<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextStatisticsDialog</class>
 <widget class="QDialog" name="TextStatisticsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditCC"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Line count</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditWC"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Character count</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Word count</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEditLC"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEditSZ"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string notr="true">MD5</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditMD5"/>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string notr="true">SHA1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditSHA1"/>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string notr="true">SHA256</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditSHA256"/>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string notr="true">SHA512</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditSHA512"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>38</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
